libvmi/os/linux/core.c

```diff
@@ -233,24 +302,47 @@ static status_t init_from_rekall_profile(vmi_instance_t vmi)
     return ret;
 }
 
+/* Is this the page holding the init task? */
 static status_t init_task_kaslr_test(vmi_instance_t vmi, addr_t page_vaddr)
 {
     status_t ret = VMI_FAILURE;
     uint32_t pid;
+    addr_t addr;
```

Comment:
worth initializing these two to a sensible value

Reply:
Will do